modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1232,6 +1232,8 @@ public void testAddSystemObjectDefinition() throws Exception {
 				ObjectDefinitionConstants.
 					EXTERNAL_REFERENCE_CODE_PREFIX_SYSTEM_OBJECT_DEFINITION));
 		Assert.assertEquals("/test", objectDefinition.getRESTContextPath());
+		Assert.assertEquals(
+			"l_testId", objectDefinition.getPKObjectFieldName());
```

Comment:
@p-albuquerque why did you put the `getPKObjectFieldName` after the `getRESTContextPath`?

Reply:
Good question